nx-X11/programs/Xserver/hw/nxagent/Clipboard.c

```diff
@@ -1583,6 +1583,9 @@ int nxagentInitClipboard(WindowPtr pWin)
     nxagentXFixesInfo.Initialized = 1;
   }
 
+  /*
+     Removing this seems to solve xg2o issue #952 (no paste directly after session start)
+
```

Comment:
Thanks for providing this patch. Please be less X2Go BTS centric in the complete PR. Describe what your patch fixes:

* in the PR name (please rename)
* in the comment above
* in the commit msg